src/themes/creator.js

```diff
@@ -412,100 +477,126 @@ export default function createTheme(
     },
     typography: {
       font100: {
-        fontFamily: primitives.primaryFontFamily,
-        fontSize: '11px',
-        fontWeight: 'normal',
-        lineHeight: '16px',
-      },
-      font200: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '12px',
         fontWeight: 'normal',
         lineHeight: '20px',
       },
-      font250: {
+      font150: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '12px',
-        fontWeight: 500,
+        fontWeight: '500',
         lineHeight: '20px',
       },
-      font300: {
+      font200: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '14px',
         fontWeight: 'normal',
         lineHeight: '20px',
       },
-      font350: {
+      font250: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '14px',
         fontWeight: 500,
         lineHeight: '20px',
       },
-      font400: {
+      font300: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '16px',
         fontWeight: 'normal',
         lineHeight: '24px',
       },
-      font450: {
+      font350: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '16px',
         fontWeight: 500,
         lineHeight: '24px',
       },
+      font400: {
+        fontFamily: primitives.primaryFontFamily,
+        fontSize: '18px',
+        fontWeight: 'normal',
+        lineHeight: '28px',
+      },
+      font450: {
+        fontFamily: primitives.primaryFontFamily,
+        fontSize: '18px',
+        fontWeight: 500,
+        lineHeight: '28px',
+      },
+      // deprecated, maintained for backward compatibility, may be removed in the future
       font460: {
```

Comment:
It doesn't make sense to keep `460` and `470` while we removing `500`, `600` and so on that is `X00 >=500`.

Reply:
yeah i agree - i think at some point there was a request to leave them in but if we are breaking it this much it may be confusing to leave those.